pkg/config/config_windows.go

```diff
@@ -23,7 +23,30 @@ func init() {
 	}
 	// NOTE: On Windows, we need at least ComSpec for the agent to be able to run batch files
 	// and SystemRoot for the integration to be able to use the networking layer in some libraries.
-	defaultPassthroughEnvironment = []string{"ComSpec", "SystemRoot"}
+	// default Windows global environment variables
+	defaultPassthroughEnvironment = []string{
```

Comment:
quick question: How did we selected this list of env variables. Would be possible to add more context in comment note?

Reply:
Good point, it was discussed in this issue: https://github.com/newrelic/infrastructure-agent/issues/1115

Would you add a comment with the issue url?